### PR TITLE
Rel v1.11.0.1

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: src/optionroms.c [hunk 1]
```diff
@@ -136,9 +136,19 @@ init_optionrom(struct rom_header *rom, u16 bdf, int isvga)
 
     tpm_option_rom(newrom, rom->size * 512);
 
+    //TODO: Find a way to hide initialisation string of iPXE
+    // It is called everytime and I have no idea how it was hidden before.
+    // For now it can be commented out and seems to work good.
```
Comment (pietrushnic) [inline src/optionroms.c]: @macpijan is it something that you implemented ? Have you got any ideas ?

**Comment by pietrushnic** [comment]
@mek-x we will need review before end of month

File: src/optionroms.c (hunk 1 above)
Comment (miczyg1) [reply src/optionroms.c]: @pietrushnic did You mean me or @macpijan ?

Comment (pietrushnic) [reply src/optionroms.c]: @miczyg1 I mean @macpijan - I'm pretty sure he did initial implementation. Git blame ?

Comment (miczyg1) [reply src/optionroms.c]: No, It just also sounded like question directed to me. Wanted to ensure myself only.

Comment (macpijan) [reply src/optionroms.c]: @pietrushnic Yes I can remember doing pxe enable/disable. @miczyg1 Which is initialization string we want to hide?

File: src/optionroms.c [hunk 2]
```diff
@@ -136,9 +136,19 @@ init_optionrom(struct rom_header *rom, u16 bdf, int isvga)
...
+
+    /*
```
Comment (mek-x) [inline src/optionroms.c]: please remove commented-out code. Git history should be enough. You can leave comment why it was changed for future reference.

File: src/optionroms.c (hunk 1 above)
Comment (miczyg1) [reply src/optionroms.c]: @macpijan the string is something like that:
```
iPXE (http://ipxe.org) 00:00.0 C000 PCI2.10 PnP PM
Press Ctrl-B to configure iPXE (PCI 00:00.0)...
```
Whenever callrom() function is called, the string is printed out on the screen/console.

**Comment by miczyg1** [comment]
@mek-x commented-out code removed

**Pull request closed without merging**